lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,77 @@
+
+        create(:event_schedule, event: first_scheduled_event, schedule: selected_schedule, start_time: conference.start_date + conference.start_hour.hours, room: conference_rooms.first)
+        create(:event_schedule, event: second_scheduled_event, schedule: selected_schedule, start_time: conference.start_date + conference.start_hour.hours + 15.minutes, room: conference_rooms.second)
+        create(:event_schedule, event: multiple_speaker_event, schedule: selected_schedule, start_time: conference.start_date + conference.start_hour.hours + 30.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: first_scheduled_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours + 15.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: second_scheduled_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours + 30.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: multiple_speaker_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours, room: conference_rooms.first)
+
+        create(:registration, user: user1, conference: conference)
+        create(:registration, user: user2, conference: conference)
+    end
+
+    # This is a full conference demo instance that has already happend.
+    # By full conference it means all basic information about conference is already set.
+    # Initially end date is set 6 days from now
+    # So that events can be created without any failure in validations.
+    conference = create(:full_conference, title: 'Jangouts Demo Conference', short_title: 'jangouts', start_date: 7.days.ago, end_date: 6.days.from_now, start_hour: 15, end_hour: 20, description: 'This is a full conference demo instance happened in the past. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.')
+    generate_program conference
+    conference.program.cfp.update_attributes!(start_date: 4.days.ago, end_date: 2.days.ago)
+    conference.update_attributes!(end_date: 1.day.ago)
+    conference.registration_period.update_attributes!(start_date: 9.days.ago, end_date: 8.days.ago)
+
+    # This is a full conference demo instance that will happen in the future.
+    # By full conference it means all basic information about conference is already set.
+    conference = create(:full_conference, title: 'Open Source Event Manager Demo Conference', short_title: 'osemdemo', start_date: 2.days.from_now, end_date: 6.days.from_now, start_hour: 8, end_hour: 20, description: 'This is a full conference demo instance happening in the future. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.')
+    generate_program conference
+
+    # This is a conference demo instance that will happen in the future.
```

Comment:
This seems to be the only conference that is to a full_conference. What about we move it either on the top or on the bottom?